packages/psss/src/api/queries/useData.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { useQuery } from 'react-query';
+import { get } from '../api';
+
+/**
+ *
+ * @param endpoint - api endpoint
+ * @param apiOptions - axios options
+ * @param queryOptions - react-query options
+ * @returns {}
+ */
+export const useData = (endpoint, apiOptions, queryOptions) => {
+  const query = useQuery([endpoint, apiOptions.params], () => get(endpoint, { ...apiOptions }), {
```

Comment:
I guess `useQuery` does not require new references to the object fields, but I may be wrong:

```suggestion
  const query = useQuery([endpoint, apiOptions.params], () => get(endpoint, apiOptions), {
```